### DISCUSSION TITLE
Any solution for state persist in Model?

### DISCUSSION THREAD

**Discussion by whoitao** [web-infra-dev/modern.js#3577] [Q&A]
How to persist my state in Model?

**Comment by botagar** [comment]
I would like options for this as well.
Currently I want to save certain models in Browser Local Storage but there doesn't seem to be any options to be able to do this natively within Reduck.
My use case is saving a Users details in a Model on sign in with an OAuth2 Auth Flow.
For those who don't know what might make this tricky is that the Auth Flow can open a new Tab where the tokens are exchanged there, and if you want to direct the user back to the original page, the model set in that other tab doesn't flow across tabs, nor does it persist across refreshes.
I created a provider that manually synchronizes user data to and from Local Storage into the Reduck Model, but I would like a more elegant way.
